php/commands/search-replace.php

```diff
@@ -52,6 +52,9 @@ class Search_Replace_Command extends WP_CLI_Command {
 	 * [--verbose]
 	 * : Prints rows to the console as they're updated.
 	 *
+	 * [--regex]
+	 * : Runs the search using a regular expression. Using --regex slows the search-replace down significantly.
```

Comment:
Can you include some more prominent "Warning: " with specific numbers as to how much it will slow things down?